wazo_acceptance/helpers/call_permission.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+class CallPermission:
+
+    def __init__(self, context):
+        self._context = context
+        self._confd_client = context.confd_client
+
+    def add_user(self, call_permission, user):
+        self._confd_client.call_permissions(call_permission).add_user(user)
```

Comment:
You can bypass the helper
Otherwise, it doesn't follow the STYLEGUIDE: https://github.com/wazo-platform/wazo-acceptance/blob/master/STYLEGUIDE.md#helper-files